pkg/catalog/manager/manager.go

```diff
@@ -100,7 +101,7 @@ func hash(content string) string {
 	return hex.EncodeToString(sum[:])
 }
 
-func (m *Manager) prepareGitRepoPath(catalog v3.Catalog) (string, string, error) {
+func (m *Manager) prepareGitRepoPath(catalog v3.Catalog, repoURL string) (string, string, error) {
```

Comment:
should we also add forgot error logic to git?

Reply:
![image](https://user-images.githubusercontent.com/4569037/48117777-361ab000-e2a5-11e8-8efe-d30f7f97872e.png)

we may not need to add forgot error logic to the git, if the username and password are incorrect the git ls-remote will return `false` and the logic will will try prepareHelmRepoPath only.